static/js/drafts.js

```diff
@@ -190,7 +193,7 @@ function maybe_notify(no_notify) {
     }
 }
 
-export function update_draft(opts = {}) {
+export function update_draft(opts = {}, sendingMessage) {
```

Comment:
Just a quick comment -- we use `_` rather than camelCase for variable names in the project.

Reply:
Changed the variable `sendingMessage` to `sending_message`.
Let me know if it requires more changes